vehicle/mkv/software/air_control/air.c

```diff
@@ -341,22 +315,22 @@ static void state_machine_run(void) {
 
             if (motor_controller_voltage < MOTOR_CONTROLLER_THRESHOLD_LOW_dV) {
                 once = true;
-                air_control_critical.air_state = IDLE;
+                air_control_critical.air_state = AIR_STATE_IDLE;
                 return;
             } else {
                 set_fault(AIR_FAULT_DISCHARGE_FAIL);
                 once = true;
                 return;
             }
         } break;
-        case FAULT: {
+        case AIR_STATE_FAULT: {
             gpio_set_pin(FAULT_LED);
             gpio_clear_pin(PRECHARGE_CTL);
             gpio_clear_pin(AIR_N_LSD);
         } break;
         default: {
             // Shouldn't happen, but just in case
```

Comment:
can you expand on what you mean by "just in case". I know you mean that we want to make sure we don't accidentally start precharge when something is wrong but I think it would be nice to include :)